lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1273,6 +1273,10 @@ def _create_node_arm_parameters(
         assert isinstance(
             capability.disk.disk_controller_type, schema.DiskControllerType
         )
+        assert (
```

Comment:
Can it be check in Azure Disk feature?

Reply:
@squirrelsc None of the other functions made sense to put this check in. Do you have a suggestion?

Reply:
How about `check` in Azure Disk options? It won't bring an exception, but skip one or all test cases.

Reply:
The check function doesn't have access to the right information, right? Does it have access to the generation and the final disk_controller_type?

Reply:
I see, it looks cross feature validation have to be here. I'm ok now.